### PR TITLE
Improve configurability of verbosity behavior

### PR DESCRIPTION
**Pull request by moui72**
As discussed in issue #29 this PR would add a configuration option to enable functionality wherein pytest-tldr behaves as it does for test runs with no verbosity, except when it prints reports after all tests are complete, where it will use whatever verbosity level was set when pytest was invoked.

This behavior would be turned on by setting `run_quiet_report_verbose` to `True`

The only other change is that I needed to include `setup_requires = pytest >= 3.5.0` to setup.cfg::options in order to build from source

### CODE REVIEW

File: pytest_tldr.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class ExitCode:
         TESTS_FAILED = EXIT_TESTSFAILED
 
 
-__version__ = '0.2.2'
+__version__ = '0.2.3b'
```
Comment (freakboy3742) [inline pytest_tldr.py:24]: The version attribute is not something a PR should be editing - it's a project management concern.

File: pytest_tldr.py [hunk 2]
```diff
@@ -60,6 +61,7 @@ def __init__(self, config, file=None):
         self.verbosity = self.config.option.verbose
         self.xdist = getattr(self.config.option, 'numprocesses', None) is not None
         self.hasmarkup = False
+        self.run_quiet_report_verbose = getattr(self.config.option, 'run_quiet_report_verbose', None)
```
Comment (freakboy3742) [inline pytest_tldr.py:64]: Is there a reason this defaults to None, rather than False?

**Review by freakboy3742 — CHANGES_REQUESTED** [review]
Apologies for the delay in responding to this. 

I've flagged a couple of small things inline, but on the whole this PR looks good. The CI is currently failing because there isn't a changelog snippet; every PR needs to include a file in the `changes` folder that describes the change for the purposes of the final changelog. In this case, the file will be called `30.feature.rst` (since it's a new feature, relating to issue #30).

I'd make the tweaks myself, but you've submitted the PR from your master branch, so I can't edit it directly.

**Comment by moui72** [comment]
thanks for finally getting to this! unfortunately, I am no longer able to spend time here, as I have changed jobs. I hope to come back to this in the future, but I'm gonna close for now.

File: pytest_tldr.py (hunk 2 above)
Comment (moui72) [reply pytest_tldr.py:64]: I think I misunderstood how options work >< I need to figure out how to register an option

File: pytest_tldr.py (hunk 1 above)
Comment (moui72) [reply pytest_tldr.py:24]: gotcha, sorry about that!

**Pull request closed without merging**